validator/iam_allowed_bindings_test.rego

```diff
@@ -16,82 +16,61 @@
 
 package templates.gcp.GCPIAMAllowedBindingsConstraintV1
 
-import data.test.fixtures.iam_allowed_bindings.assets as fixture_assets
-import data.test.fixtures.iam_allowed_bindings.constraints as fixture_constraints
-
-# Find all violations on our test cases
-find_violations[violation] {
-	asset := fixture_assets[_]
-	constraint := data.test_constraints[_]
-
-	issues := deny with input.asset as asset
-		 with input.constraint as constraint
-
-	total_issues := count(issues)
+template_name := "GCPIAMAllowedBindingsConstraintV1"
 
-	violation := issues[_]
-}
-
-blacklist_role_violations[violation] {
-	constraints := [fixture_constraints.iam_allowed_bindings_blacklist_role]
-
-	found_violations := find_violations with data.test_constraints as constraints
+import data.validator.test_utils as test_utils
 
-	violation := found_violations[_]
-}
+import data.test.fixtures.iam_allowed_bindings.assets as fixture_assets
+import data.test.fixtures.iam_allowed_bindings.constraints as fixture_constraints
 
-test_blacklist_role_violations {
-	count(blacklist_role_violations) = 2
+# Test blacklist project
+test_blacklist_project_violation_count {
+	test_utils.check_test_violations_count(fixture_assets, [fixture_constraints.iam_allowed_bindings_blacklist_project], template_name, 1)
 }
 
-blacklist_project_violations[violation] {
-	constraints := [fixture_constraints.iam_allowed_bindings_blacklist_project]
-
-	found_violations := find_violations with data.test_constraints as constraints
-
-	violation := found_violations[_]
+# Test blacklist public
+test_blacklist_role_violation_count {
+	test_utils.check_test_violations_count(fixture_assets, [fixture_constraints.iam_allowed_bindings_blacklist_public], template_name, 2)
 }
 
-test_blacklist_project_violations {
-	count(blacklist_project_violations) = 1
+# Test blacklist role
```

Comment:
Duplicate test from line 26?

Reply:
Thanks for catching, the methods were duplicated but the tests were not. Update the names to reflect what is actually being tested.